dashboard/db/migrate/20240501215821_create_cap_user_events.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCapUserEvents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :cap_user_events do |t|
```

Comment:
I'm on the fence but slightly leaning towards the shorter name because ChildAccountPolicyUserEvents is approaching a length which makes me uncomfortable (I acknowledge this is totally just a gut feeling). I think it should be pretty easy to discover what "Cap" stands for given it's usage in the code base.